lib/spack/spack/package.py

```diff
@@ -2601,6 +2636,9 @@ def _run_default_build_time_test_callbacks(self):
         if self.build_time_test_callbacks is None:
             return
 
+        # Log running install-time tests to the build log
+        tty.msg('Running install-time tests')
+
```

Comment:
Message is in the wrong method

Reply:
Actually, that was a copy-and-paste issue.  I meant but forgot to change it.